### DISCUSSION TITLE
induction hob

### DISCUSSION THREAD

**Discussion by FlorinFrance** [iklein99/homebridge-smartthings#143] [Q&A]
hello,
did you know if SAMSUNG induction hob is showing at least on/off in homekit? ( need it for automation)

thx in advance !

**Comment by iklein99** [comment]
I don’t know.  Is that a cook too or oven?  Anyway easiest way to find out is to add the plug-in, restart the homebridge and see what it looks like b. If it supports a switch capability then it should work.  Sent from my iPhoneOn Feb 26, 2023, at 2:27 AM, FlorinFrance ***@***.***> wrote:﻿
hello,
did you know if SAMSUNG induction hob is showing at least on/off in homekit? ( need it for automation)
thx in advance !

**Reply by FlorinFrance** [reply]
hello, is a cooktop; i want to buy if at least on/off is working.
thx

**Reply by iklein99** [reply]
Hi.  I can't tell you how this will look in Home Kit, but my guess is that it will show up at least as a switch.  The plugin looks at the CAPABILITIES reported by the device and supports a number of them including SWITCH.  So if the device supports this capability then it will work.  But since I don't have the device, I can't test it for you.  If you run into problems, I will look at the logs for you to see what the device reports.

**Reply by FlorinFrance** [reply]
thank you sir !
